data/transition-sites/gds_alphagov_content-planner-preview.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: gds_alphagov_content-planner-preview
```

Comment:
Yep, I'll remove it from this PR (this PR was originally created before content-planner was removed in https://github.com/alphagov/transition-config/pull/1128).